sky/utils/schemas.py

```diff
@@ -4,11 +4,11 @@
 https://json-schema.org/
 """
 
-from sky.clouds import cloud_registry
-from sky.data import storage
-
 
 def get_resources_schema():
+    # To avoid circular imports, only import when needed.
+    # pylint: disable=import-outside-toplevel
+    from sky.clouds import service_catalog
     return {
         '$schema': 'http://json-schema.org/draft-07/schema#',
```

Comment:
For code gardening: why do we use a different schema for resources than the others?

Reply:
Hm, I'm not sure actually. I've updated all of the schemas to the newest one to be consistent, and tested that it still works as expected.